### DISCUSSION TITLE
v0.7.1

### DISCUSSION THREAD

**Discussion by trollixx** [zealdocs/zeal#1627] [Announcements]
[Full Changelog](https://github.com/zealdocs/zeal/compare/v0.7.0...v0.7.1) | [Closed Issues](https://github.com/zealdocs/zeal/milestone/16?closed=1)

A bugfix release for the 0.7 branch.

### Fixes

* **browser:** send key events to web view's focus proxy ([f2212eb](https://github.com/trollixx/zeal/commit/f2212eb4f0c9b39a0a207995b097a6b0238b3cbb)), closes [#703](https://github.com/trollixx/zeal/issues/703)
* **registry:** save non-zero docset revision in meta.json ([2cefcd4](https://github.com/trollixx/zeal/commit/2cefcd45423e58283372a4c735609c284eded410)), closes [#1532](https://github.com/trollixx/zeal/issues/1532) [#1559](https://github.com/trollixx/zeal/issues/1559)
* **ui:** add tool tip when global shortcuts are not supported ([1d92d3c](https://github.com/trollixx/zeal/commit/1d92d3c737f04e9f7ba369777495a1fa4178ddd4))
* **ui:** use async selection in search edit ([d5ce8a0](https://github.com/trollixx/zeal/commit/d5ce8a0f04dfc4d609cfcab75cc53135ba1a9557)), closes [#1033](https://github.com/trollixx/zeal/issues/1033) [#703](https://github.com/trollixx/zeal/issues/703)

### Build System

* **cmake:** find libarchive from brew on macos ([#1621](https://github.com/trollixx/zeal/issues/1621)) ([1ff2d6a](https://github.com/trollixx/zeal/commit/1ff2d6ab3e38f73ef75aac40384fc73c6745bff3))

### CI/CD

* **github:** build with Qt 6.6.3 ([cbd3208](https://github.com/trollixx/zeal/commit/cbd3208aca598c180b17ffcb3937c4eb3b8adf7f))
* **github:** use appstreamcli to validate Linux metadata file ([de213a7](https://github.com/trollixx/zeal/commit/de213a74d8bd45b40638dd6d583ef82f819c14c0))
* **github:** validate AppStream metadata ([46e18c2](https://github.com/trollixx/zeal/commit/46e18c2acb2579d172c78bad6778b19c64eaa470))

---

### New Contributors

* @NickCao made their first contribution in https://github.com/zealdocs/zeal/pull/1565
* @guihkx made their first contribution in https://github.com/zealdocs/zeal/pull/1566
* @zchrissirhcz made their first contribution in https://github.com/zealdocs/zeal/pull/1621

<hr /><em>This discussion was created from the release <a href='https://github.com/zealdocs/zeal/releases/tag/v0.7.1'>v0.7.1</a>.</em>

**Comment by foxik** [comment]
> Please don't hesitate to ping me in the future when there is a new version, so that I update the official Debian package. I normally see new versions on [my dashboard](https://qa.debian.org/developer.php?login=mquinson&comaint=yes), but I don't check it every day.

@mquinson Pinging that a new bugfix version of Zeal was released, following https://github.com/orgs/zealdocs/discussions/1541#discussioncomment-7183393 :bow:

**Reply by mquinson** [reply]
Thanks for the reminder, I totally forgot about that. The 0.7.1 is now uploaded to the build servers. Please do not hesitate to ping me in the future.
